### PR TITLE
css-pseudo-classes Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
animation5

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+button {
+  font-size: larger;
+  padding: 10px 25px;
+  text-align: center;
+  border: 1px solid gray;
+}
+
+button:hover {
```
Comment (robgmerrill) [inline css-pseudo-classes/styles.css:8]: We want the cursor to be set to pointer on hover.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Close! One small change regarding the cursor on button hover.

**Review by robgmerrill — APPROVED** [review]
👍

**Comment by Nathan-Chon** [comment]
animation7

**Pull request merged**